crates/sui/tests/protocol_version_tests.rs

```diff
@@ -145,16 +145,21 @@ mod sim_only_tests {
 
         timeout(Duration::from_secs(60), async move {
             while let Ok(committee) = epoch_rx.recv().await {
+                let protocol_version = test_cluster
+                    .fullnode_handle
+                    .sui_node
+                    .state()
+                    .epoch_store_for_testing()
+                    .protocol_version();
```

Comment:
this is not guaranteed to work - the fullnode could reconfigure again by the time the message is received.

but that's probably unlikely - so I think i'm ok with this if the tests pass reliably (`MSIM_TEST_NUM=20`). However it shouldn't be very hard to just have the epoch change receiver send the new protocol version as well.

Reply:
Updated the channel